Orange/widgets/visualize/owscatterplot.py

```diff
@@ -322,9 +322,7 @@ def set_data(self, data):
             self.init_attr_values()
         self.vizrank.initialize()
         self.vizrank.attrs = self.data.domain.attributes if self.data is not None else []
-        self.vizrank_button.setEnabled(
-            self.data is not None and self.data.domain.class_var is not None
-            and len(self.data.domain.attributes) > 1 and len(self.data) > 1)
+        self._vizrank_button_enabled()
```

Comment:
Well, I moved code somewhere else. Now I have put it back here and added check for sparse.